utility/build.gradle

```diff
@@ -43,7 +43,8 @@ dependencies {
       'androidx.appcompat:appcompat:1.0.2',
       'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03',
       'com.google.dagger:dagger:2.24',
-      "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+      "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
+      'com.github.bumptech.glide:glide:4.9.0',
```

Comment:
Prefer maintaining the alphabetical order.

Reply:
corrected.

Reply:
Still not in alphabetical order.

Reply:
Changed to alphabetical order.